### PR TITLE
[mesheryctl] fix gloabl help message

### LINKED ISSUE #2766: [mesheryctl] Global help: change "performance testing" to "performance management"
_See [mesheryctl Command Tracker](https://bit.ly/3dqXy1q) for current status of commands._

#### Current Behavior
The default help text in `mesheryctl` uses the phrase "performance testing". 

#### Desired Behavior

The project has graduated from this and should use the phrase "Performance Management" instead.

#### Screenshots/Logs
```
➜  ~ mesheryctl
Meshery is the service mesh management plane, providing lifecycle, performance, and configuration management of service meshes and their workloads.

Usage:
  mesheryctl [command]

Available Commands:
  exp         Experimental commands for mesheryctl
  help        Help about any command
  mesh        Service Mesh Lifecycle Management
  pattern     Meshery Patterns Management
  perf        Performance Testing
  system      Meshery Lifecycle Management
  version     Version of mesheryctl
```

#### Environment

- Meshery Version: v0.5.2

#### Contributor Resources

- [mesheryctl Contributing Guide](https://github.com/layer5io/meshery/blob/master/mesheryctl/README.md)
- [Beginner's guide to contributing to Meshery and mesheryctl](https://youtu.be/hh_kFLZx3G4)
- [mesheryctl Command Tracker](https://docs.google.com/spreadsheets/d/1q63sIGAuCnIeDs8PeM-0BAkNj8BBgPUXhLbe1Y-318o/edit#gid=0)
- [Meshery CLI Commands and Documentation](https://docs.google.com/document/d/1xRlFpElRmybJ3WacgPKXgCSiQ2poJl3iCCV1dAalf0k/edit#heading=h.5fucij4hc5wt)

### PR DESCRIPTION
**Pull request by fallmor**
**Description**

This PR fixes #2766

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by leecalcote** [comment]
@fallmor, excellent. 😄 

A small item to look after is sign-off(s) on your commit(s). Here are one-time instructions to execute - https://github.com/layer5io/meshery/pull/2772/checks?check_run_id=2471452578

**Comment by leecalcote** [comment]
...and here are instructions to configure your environment to do it implicitly - https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin

**Comment by leecalcote** [comment]
@fallmor, if you will take a moment to add the issue # in the PR description ```This PR fixes #```, that would be great. The issue will be auto-closed when the PR merges in this case.

**Comment by fallmor** [comment]
thank you @leecalcote i execute the sign-off command but it does noy seem to take effect

**Comment by leecalcote** [comment]
Hmm. Weird. Ok. Let me manually flag this one as a pass.

File: mesheryctl/internal/cli/root/perf/perf.go [hunk 1]
```diff
@@ -47,8 +47,8 @@ var (
 // PerfCmd represents the Performance Management CLI command
 var PerfCmd = &cobra.Command{
 	Use:     "perf",
-	Short:   "Performance Testing",
-	Long:    `Performance Testing & Benchmarking using Meshery CLI.`,
+	Short:   "Performance Management",
+	Long:    `Performance Management & Benchmarking using Meshery CLI.`,
```
Comment (Naimaoye) [inline mesheryctl/internal/cli/root/perf/perf.go:51]: Good job @fallmor . Well done!

**Review by Naimaoye — APPROVED** [review]

**Review by navendu-pottekkat — APPROVED** [review]
LGTM

**Pull request merged**